packages/matchbox/src/components/Page/Page.js

```diff
@@ -144,7 +163,14 @@ function Page(props) {
               </Box>
             )}
           </Box>
-          <Box flex="0" display="flex">
+          <Box
```

Comment:
Hmm not sure if this can be done easily, should the subtitle be before the actions?